src/main/java/com/infobip/api/SmsApi.java

```diff
@@ -147,9 +147,14 @@ public GetInboundSmsMessagesRequest getInboundSmsMessages() {
         return new GetInboundSmsMessagesRequest();
     }
 
-    private RequestDefinition getOutboundSmsMessageDeliveryReportsDefinition(
-            String bulkId, String messageId, Integer limit, String applicationId, String entityId) {
-        RequestDefinition.Builder builder = RequestDefinition.builder("GET", "/sms/1/reports")
+    private RequestDefinition getOutboundSmsMessageDeliveryReportsV3Definition(
```

Comment:
Do we need to include 'V3' in these new classes? Client already has information about which version of endpoints are available in given release, and we might ease off transition to this new version since naming would stay the same.